tests/connection.js

```diff
@@ -285,6 +285,7 @@ exports.relaying = {
     'sets and gets' (test) {
         test.expect(3);
         test.equal(this.connection.relaying, false);
+        // FIXME: should this be assigned vs compared?
```

Comment:
Doesn't the next test answer the question? Try changing it and see what happens to the test. Poke at it until your satisfied and then remove this comment.